exporters/logging/src/main/java/io/opentelemetry/exporter/logging/LoggingSpanExporter.java

```diff
@@ -18,6 +18,9 @@
 public final class LoggingSpanExporter implements SpanExporter {
   private static final Logger logger = Logger.getLogger(LoggingSpanExporter.class.getName());
 
+  /** Class constructor. */
+  public LoggingSpanExporter() {}
```

Comment:
Noticed that there are public constructors in `:exporters:logging`. Thought about deprecating them, and adding `public static LoggingSpanExporter create()` for symmetry, but figured that would just be clutter since we can never actually get rid of the public constructor. 

Instead, I opted to at least make the public constructors explicit.

Reply:
Doh - I think it's fine to add the create and deprecate to at least show the consistent pattern within our codebase.